routers/user.controller.js

```diff
@@ -85,14 +95,39 @@ class UserController {
       }).validate(req.body);
 
       if (validationError) {
-        throw new ApiError(400, 'Bad requiest', validationError);
+        throw new ApiError(400, 'Bad request', validationError);
       }
 
       next();
     } catch (e) {
       errorHandler(req, res, e);
     }
   }
+
+  //========================================
+
+  async logout(req, res) {
+    try {
+      const { activeToken, user } = req;
+
+      user.update(user.id, { $pull: { tokens: { activeToken } } });
```

Comment:
Have you checked if it's working or not? Pretty sure that it should be `tokens: activeToken` instead